app/models/katello/pool.rb

```diff
@@ -2,14 +2,22 @@ module Katello
   class Pool < Katello::Model
     include Katello::Authorization::Pool
 
-    attr_accessor :quantity_attached
+    def quantity_attached
+      if activation_keys.present?
+        activation_keys.first.get_key_pools
+          .select { |pool| pool['id'] == cp_id }.first.try(:[], 'amount')
+      end
+    end
```

Comment:
This works fine in the context of our API, where we are returning the quantity attached for a given subscription for one activation key. But if one runs something like ```my_pool.quantity_attached``` it would only return the quantity attached for the first activation key, which isn't obvious. The pool could have many activation keys using it. Is there a way to specify the activation key in the method, and return the first one as default?